payments-model/src/main/java/com/stripe/android/repository/ConsumersApiService.kt

```diff
@@ -71,6 +81,41 @@ class ConsumersApiServiceImpl(
         )
     }
 
+    /**
+     * Triggers an SMS verification for the consumer corresponding to the given client secret.
+     */
+    override suspend fun startConsumerVerification(
+        consumerSessionClientSecret: String,
+        locale: Locale,
+        authSessionCookie: String?,
+        requestOptions: ApiRequest.Options
+    ): ConsumerSession {
+        return executeRequestWithModelJsonParser(
+            stripeErrorJsonParser = stripeErrorJsonParser,
+            stripeNetworkClient = stripeNetworkClient,
+            request = apiRequestFactory.createPost(
+                startConsumerVerificationUrl,
+                requestOptions,
+                mapOf(
+                    "request_surface" to "android_payment_element",
```

Comment:
Is this line still accurate for your use case? Can/should we do this dynamically?

Reply:
Great catch. This also applies to the `lookup` endpoint. Connections will be using a different surface here (TBD)

https://github.com/stripe/stripe-android/pull/6132/commits/5122691fd98182b92977ca3241e99fb5ee4386bb